custom_directives.py

```diff
@@ -25,7 +25,7 @@ def run(self):
 
         try:
             text = open(filename).read()
-            text_no_docstring = self.docstring_regex.sub('', text)
+            text_no_docstring = self.docstring_regex.sub('', text, count = 0)
```

Comment:
Figured it! `count=1` is the right parameters. (no spaces around `=` because pep8)
Sorry that I've said it's count=0 in the issue.